sip-test-kit/src/test/java/de/ikor/sip/foundation/testkit/config/AutoTestCaseLoadingTest.java

```diff
@@ -8,20 +8,25 @@
 import org.springframework.core.env.ConfigurableEnvironment;
 
 @SpringBootTest(
-    properties = "sip.testkit.test-cases-path:test-tcd.yml",
+    properties = {"sip.testkit.test-cases-path:test-tcd.yml", "sip.testkit.batchTest:true"},
     classes = ConfigurableEnvironment.class)
 class AutoTestCaseLoadingTest {
+
+  private static final String ROUTE_CONTROLLER_SUPERVISE =
+      "camel.springboot.routeControllerSuperviseEnabled";
+
   @Autowired private ConfigurableEnvironment environment;
 
   @Test
-  void When_testKitTests_Expect_PropertiesFromFileInEnv() {
+  void GIVEN_necessaryInputProperties_WHEN_testKitTests_THEN_getExpectedProperties() {
     // arrange
-    AutoTestCaseLoading autoTestCaseLoading = new AutoTestCaseLoading();
+    AutoTestCaseLoading subject = new AutoTestCaseLoading();
 
     // act
-    autoTestCaseLoading.testKitTests(environment);
+    subject.testKitTests(environment);
```

Comment:
testKitTests seams like odd method name in this case, don't you think?
I don't remember why we call it so. Would it make sense to refactor it?

Reply:
Method name changed to "prepareTestingEnvironment". Bean name is little bit enhanced (testKitBatchTests instead of testKitTests).